tiledb/sm/cpp_api/config.h

```diff
@@ -892,6 +870,12 @@ class Config {
    * - `config.logging_format` <br>
    *    The logging format configured (DEFAULT or JSON)
    *    **Default**: "DEFAULT"
+   * - `profile_name` <br>
+   *    The name of the Profile to be used for REST transactions. <br>
```

Comment:
Nit: "used for REST transactions" could be replaced by a more specific phrase, I think. Something like "the name of the TileDB client configuration profile". On the Cloud team we're trying to avoid "REST" to prevent confusion with the legacy system.

Reply:
"transactions" is a very weighted term in the database world so I'd also suggest finding a different term for that.  Requests?  Sessions?